assets/src/application/ui/layout/confirmDelete/ConfirmDelete.tsx

```diff
@@ -1,19 +1,56 @@
 import React from 'react';
-import { Popconfirm } from 'antd';
-import { PopconfirmProps } from 'antd/lib/popconfirm';
+import {
+	AlertDialog,
+	AlertDialogBody,
+	AlertDialogFooter,
+	AlertDialogHeader,
+	AlertDialogContent,
+	AlertDialogOverlay,
+	Button,
+} from '@chakra-ui/core';
+import { EspressoButton, EspressoButtonProps } from '@application/ui/input';
 import { __ } from '@wordpress/i18n';
 
-interface ConfirmDeleteProps extends Omit<PopconfirmProps, 'title'> {
+interface ConfirmDeleteProps {
+	buttonProps: EspressoButtonProps;
+	onConfirm: VoidFunction;
 	title?: string;
 }
 
-const ConfirmDelete: React.FC<ConfirmDeleteProps> = ({ children, ...props }) => {
+const ConfirmDelete: React.FC<ConfirmDeleteProps> = ({ buttonProps, onConfirm, ...props }) => {
```

Comment:
![Peek 2020-04-08 23-06](https://user-images.githubusercontent.com/1477453/78828670-bc34ac00-79ed-11ea-9fc0-24f3d2d72146.gif)

Reply:
The other confirm component is in this PR: https://github.com/eventespresso/event-espresso-core/pull/2636/files#diff-c9c1763fc52e041df1c8a1448b30f578R7
***
since that depends on new models being implemented, the tightening up will happen in that PR